website/src/content/concepts/merchant-center-api.mdx

```diff
@@ -91,6 +101,26 @@ When you develop a Custom Application all authentication logic is handled implic
 
 </Info>
 
+# HTTP headers
+
+Accessing the Merchant Center API requires to include certain HTTP headers in the HTTP request.
+
+- `Accept`: Required.<br/>
```

Comment:
nitpick: Just a personal suggestion to use another format where description of the header is in the same line:
![image](https://user-images.githubusercontent.com/97907068/182593315-ee7fb4c4-8743-40d3-9201-5ed512042ebf.png)